src/DataStore.ts

```diff
@@ -31,7 +32,7 @@ export interface JoinConfig {
 /**
  * Sends a voice state update to the main websocket shard of a guild, to indicate joining/leaving/moving across
  * voice channels
- * @param voiceChannel The voice channel to move to
+ * @param config
```

Comment:
Missing parameter description, I was thinking of this one:
```suggestion
 * @param config The voice state data to send to Discord
```